index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/reset.css">
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="styles/products.css">
+    <title>Frog Hats</title>
+</head>
+
+<body>
+    <main>
+        <section class="main-section">
+            <img class="main-logo" src="https://img.itch.zone/aW1nLzI5NzU2ODYuZ2lm/347x500m/3N9qDN.gif"/>
+            <center>
```

Comment:
<center> isn't really what you want, this has been defunct for a few years. (you'll do this in CSS)